tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_suppliers.feature

```diff
@@ -0,0 +1,107 @@
+      | product1MWhite | Size - M, Color - White | [Size:M,Color:White] | 0               | 0           | 0        | false      |
+      | product1MBlack | Size - M, Color - Black | [Size:M,Color:Black] | 0               | 0           | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  | [Size:M,Color:Blue]  | 0               | 0           | 0        | false      |
+    And combination "product1SWhite" should not have any suppliers assigned
+    And combination "product1SBlack" should not have any suppliers assigned
+    And combination "product1Blue" should not have any suppliers assigned
+    And combination "product1MWhite" should not have any suppliers assigned
+    And combination "product1MBlack" should not have any suppliers assigned
+    And combination "product1MBlue" should not have any suppliers assigned
+    When I set following suppliers for combination "product1SWhite":
+      | reference               | supplier reference | combination supplier reference | currency | price tax excluded |
+      | product1SWhiteSupplier1 | supplier1          | sup white shirt S 1            | USD      | 10                 |
+      | product1SWhiteSupplier2 | supplier2          | sup S2                         | USD      | 0                  |
+      | product1SWhiteSupplier3 | supplier3          | sup S3                         | USD      | 5.5                |
+    Then combination "product1SWhite" should have following suppliers:
+      | combination supplier reference | currency | price tax excluded |
+      | sup white shirt S 1            | USD      | 10                 |
+      | sup S2                         | USD      | 0                  |
+      | sup S3                         | USD      | 5.5                |
+    And combination "product1SBlack" should not have any suppliers assigned
+    And combination "product1Blue" should not have any suppliers assigned
+    And combination "product1MWhite" should not have any suppliers assigned
+    And combination "product1MBlack" should not have any suppliers assigned
+    And combination "product1MBlue" should not have any suppliers assigned
+
+  Scenario: Remove all associated combination suppliers
+    Given product product1 type should be combination
+    And combination "product1SWhite" should have following suppliers:
+      | combination supplier reference | currency | price tax excluded |
+      | sup white shirt S 1            | USD      | 10                 |
+      | sup S2                         | USD      | 0                  |
+      | sup S3                         | USD      | 5.5                |
+    When I remove all associated combination "product1SWhite" suppliers
+    And combination "product1SWhite" should not have any suppliers assigned
+    And product product1 should not have a default supplier
+    And product product1 default supplier reference should be empty
```

Comment:
Can you add some tests for invalid types (once a dedicated exception will be created) For both scenarios:
- forbidden combination action on regular product (I guess it will be in the other feature file)
- forbidden regular action on combination product